README.md

```diff
@@ -3,7 +3,7 @@
 # The Bookstore
 
 > A single page bookstore application where the user
-> can add and delete books!
+> can add and delete books from an API!
```

Comment:
- [ ] Upon starting your app it does not successfully fetch and so does not render the books from the app. Kindly fix this issue.

  ![josevu-2](https://user-images.githubusercontent.com/81468675/175073821-6ae72c46-fb3e-49b9-9b79-3d4eaf6b8d15.jpg)

- [ ] I am unable to add a book to the app. When I add the book details and click the add button, the app crashes, goes blank, and returns an error. Kindly fix this.

  ![josevu-1](https://user-images.githubusercontent.com/81468675/175074073-2bd83bcf-f116-4dad-8f79-baf8a956946f.jpg)

Reply:
- [ ] [optional] Kindly update your link to your project demo, currently it has the issues highlighted by the previous reviewer.